stacks/apps/dovetail-router.yml

```diff
@@ -843,6 +843,8 @@ Resources:
               ValueFrom: !Sub /prx/${EnvironmentTypeAbbreviation}/Spire/Dovetail-Router/augury-token
             - Name: FEEDER_TOKEN
               ValueFrom: !Sub /prx/${EnvironmentTypeAbbreviation}/Spire/Dovetail-Router/feeder-token
+            - Name: TRAFFIC_SECRET
+              ValueFrom: /prx/global/Spire/Dovetail-Router/traffic-secret
```

Comment:
Good enough name?  As there's only 1 traffic lambda, this needs to be shared in every environment and region.

Reply:
I think that's good!